misc/path_utils.c

```diff
@@ -295,7 +298,7 @@ void mp_mkdirp(const char *dir)
         if (cdir)
             *cdir = 0;
 
-        mkdir(path, 0700);
+        mkdir(path, 0777);
```

Comment:
I think it was done for privacy reasons, to have this folder only visible/accessible for owner. Is there a reason for this change?